lib/bundler/cli/inject.rb

```diff
@@ -1,12 +1,14 @@
 # frozen_string_literal: true
+require "bundler/cli/common"
 module Bundler
   class CLI::Inject
-    attr_reader :options, :name, :version, :gems
-    def initialize(options, name, version, gems)
+    attr_reader :options, :name, :version, :gems, :command
+    def initialize(options, name, version, gems, command = "inject")
```

Comment:
Why can't we pass the intention to have timestamps printed as part of the options parameter instead of specifying which command is being invoked? It seems the only reason that command is being specified as `inject` or `add` is to determine whether add :remove_timestamp to the options hash or not.